### PR TITLE
Feature/gdv 455 implement stac dataset

### PR DESCRIPTION
**Pull request by CindyvdVries**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,7 @@
+VUE_APP_MAPBOX_TOKEN=
```
Comment (SiggyF) [inline .env.example:1]: Please include instructions where to obtain these values.

File: travis.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+language: node_js
```
Comment (SiggyF) [inline travis.yml:1]: I think we moved to github actions

**Review by SiggyF — APPROVED** [review]
Looks good. Can you summarize the main changes in the issue?

File: src/components/TimeStamp.vue [hunk 3]
```diff
@@ -4,7 +4,7 @@
       Date raster layer
     </v-card-title>
     <time-slider
-      :dates="activeRasterData.imageTimeseries"
+      :dates="timeseriesItems"
```
[the remark this answers is not in the document]
Comment (CindyvdVries) [reply src/components/TimeStamp.vue:7]: items - for the selection menu i'm using, so it's more timestamp items. Changing the name :)

File: travis.yml (hunk 2 above)
Comment (CindyvdVries) [reply travis.yml:1]: yes, no one told me when doing this update. will remove
